src/index.ts

```diff
@@ -61,7 +61,7 @@ const startIngestor = async (): Promise<void> => {
   /*
    * Setup all listeners we care about
    */
-  await blockListener();
```

Comment:
Not saying it's something wrong with this, I'm just curious on why you removed it

Reply:
This one's not an async function